code/game/objects/items.dm

```diff
@@ -108,6 +108,10 @@ var/global/image/fire_overlay = image("icon" = 'icons/effects/fire.dmi', "icon_s
 	..()
 	for(var/path in actions_types)
 		new path(src)
+	if(SSaltsprites)
+		for(var/datum/sprite_override/O in SSaltsprites.sprite_overrides)

```

Comment:
Type cache is new to me actually. I was gonna ask about it when I get home, unless it's initial meme